src/Mod/Draft/DraftEdit.py

```diff
@@ -252,15 +252,18 @@ def __init__(self):
                                 "Part", "Part::Line", "Part::Box"]
 
     def GetResources(self):
+        
+        tooltip = "Edits the active object.\n"\
+                  "Press E or ALT+LeftClick to display context menu\n"\
+                  "on supported nodes and on supported objects."
```

Comment:
Just a tip. The Python way to concatenate strings is to place the strings inside parentheses. Then there is automatic line continuation, string concatenation, and you don't need to use backslashes to continue the line.
```py
tooltip = ("Edits the active object.\n"
           "Press E or ALT+LeftClick to display context menu\n"
           "on supported nodes and on supported objects.")
```

Reply:
thx! done